### DISCUSSION TITLE
how to import *.wmf file

### DISCUSSION THREAD

**Discussion by echolove38** [SVG-Edit/svgedit#716] [New feature request]
I need import wmf file, but it seems as if not support.
will u make this work

**Comment by jfhenon** [comment]
Hello, This is a good suggestion. WMF is basically a vector based format so it could be possible to do the conversion. However, we have no time to do this. 
I suggest 2 directions:
1. Submitting a PR (as an extension) and we will consider it.
2. Using an online converter to do the conversion and then import the SVG into SVG-Edit. 
Thanks

**Reply by echolove38** [reply]
well, how to submit a PR ?
